app/views/pledges/index.html.erb

```diff
@@ -1,5 +1,10 @@
 <div class="container">
-  <p id="notice"><%= notice %></p>
+  <% if notice %>
```

Comment:
You have this logic in every view. Have you considered pulling out all the common elements into the layout? It'll mean less code, and (more importantly) it'll mean that we don't need to worry about consistency when we add new views (or change existing ones):

For example, in the application layout:

```erb
<div class="container">
  <%= render 'notice' if notice %>
  <%= yield %>
</div>

<%# And then a "notice" partial with the notice code %>
```